.zenodo.json

```diff
@@ -0,0 +1,29 @@
+{
+    "description": "<p>This repository contains a collection of utilities and integration tools which enhance the basic <a href=\"https://github.com/SimpleITK/SimpleITK\">SimpleITK</a> Python package functionality. Among other things, it enables data exchange between SimpleITK and other packages (e.g. <a href=\"https://vtk.org/\">vtk</a>, <a href=\"https://www.dask.org/\">dask</a>) and provides additional useful capabilities such as handling SimpleITK messages using standard Python logging.</p>",
+    "keywords": [
+        "Open Science",
+        "Open Source Software",
+        "Image Analysis",
+        "SimpleITK"
+    ],
+    "license": "Apache-2.0",
+    "title": "SimpleITK Imaris Extensions",
+    "upload_type": "software",
```

Comment:
Why is the title "SimpleITK Imaris Extensions"?

Reply:
Good catch. That's the original file from which I used to create this one from. Will correct.